ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424_model_serving_bias_metrics.robot

```diff
@@ -0,0 +1,186 @@
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource            ../../../Resources/Page/OCPDashboard/Monitoring/Metrics.robot
+Library             OpenShiftLibrary
+Suite Setup         Bias Metrics Suite Setup
+Suite Teardown      Bias Metrics Suite Teardown
+
+*** Variables ***
+${DEFAULT_MONITORING_NS}=            openshift-user-workload-monitoring
+${PRJ_TITLE}=                        model-serving-project
+${PRJ_TITLE1}=                       model-project-test
+${MODEL_ALPHA}=                      demo-loan-nn-onnx-alpha
+${MODEL_PATH_ALPHA}=                 trusty/loan_model_alpha.onnx
+${MODEL_BETA}=                       demo-loan-nn-onnx-beta
+${MODEL_PATH_BETA}=                  trusty/loan_model_beta.onnx
+${TRUSTYAI_RESOURCEPATH}=            ods_ci/tests/Resources/Files/TrustyAI
+${TRUSTYAI_CR_FILEPATH}=             ${TRUSTYAI_RESOURCEPATH}/trustyai_crd.yaml
+${MONITORING_CONFIG_FILEPATH}=       ods_ci/tests/Resources/Files/uwm_cm_conf.yaml
+${UWM_CONFIG_FILEPATH}=              ods_ci/tests/Resources/Files/uwm_cm_enable.yaml
+${aws_bucket}=                       rhods-public
+${RUNTIME_NAME}=                     Model Bias Serving Test
+${PRJ_DESCRIPTION}=                  Model Bias Project Description
+${PRJ_DESCRIPTION1}=                 Model Bias Project Description 1
+${framework_onnx}=                   onnx - 1
+
+*** Test Cases ***
+Verify DIR Bias Metrics Available In CLI For Models Deployed Prior To Enabling Trusty Service For Admin User
+    [Documentation]    Verifies that the Bias metrics are available in Metrics Console for a model which was
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    Smoke
+    ...       Tier1   ODS-2482    ODS-2479
+    [Teardown]    Delete Data Science Project From CLI    displayed_name=${PRJ_TITLE}
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${aws_bucket}
+    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}
+    Open Model Serving Home Page
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_ALPHA}    framework=${framework_onnx}    existing_data_connection=${TRUE}
+    ...    data_connection_name=model-serving-connection    model_path=${MODEL_PATH_ALPHA}    model_server=${RUNTIME_NAME}
+    ${runtime_name} =    Replace String Using Regexp    string=${RUNTIME_NAME}    pattern=\\s    replace_with=-
+    ${runtime_name} =    Convert To Lower Case    ${runtime_name}
+    Wait For Pods To Be Ready    label_selector=name=modelmesh-serving-${runtime_name}    namespace=${PRJ_TITLE}
+    Verify Model Status    ${MODEL_ALPHA}    success
```

Comment:
could you use the same model for both the test?

Reply:
Using different models beacuse one of the models is biased. Trying with different models to see how bias metrics respond to skewed metrics.